Makefile

```diff
@@ -38,18 +38,18 @@ docker-build:
 	@echo "--> Compiling the project, inside a temporary container"
 	$(eval IMAGE=$(shell uuidgen))
 	${CONTAINER_TOOL} build --target build-env -t ${IMAGE} .
-	${CONTAINER_TOOL} run --rm ${IMAGE} /bin/cat /louketo-proxy > bin/louketo-proxy
+	${CONTAINER_TOOL} run --rm ${IMAGE} /bin/cat /gatekeeper > bin/gatekeeper
```

Comment:
Let's add a `mkdir -p bin/` before this to make sure the directory exists before building

Reply:
done